### PR TITLE
Display lifted primary constructor parameters in watch window …

### PR DESCRIPTION
**Pull request by tmat**
…as children of the object

Fixes https://github.com/dotnet/roslyn/issues/74082

### CODE REVIEW

**Comment by tmat** [comment]
@cston ptal

**Comment by AlekseyTs** [comment]
@tmat It looks like there are test failures

**Comment by tmat** [comment]
@AlekseyTs fixed

File: src/ExpressionEvaluator/VisualBasic/Source/ResultProvider/VisualBasicResultProvider.vb [hunk 1]
```diff
@@ -2,6 +2,8 @@
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
 
+Imports System.Reflection.Adds
+Imports System.Runtime.InteropServices
```
Comment (cston) [inline src/ExpressionEvaluator/VisualBasic/Source/ResultProvider/VisualBasicResultProvider.vb:6]: Remove if not needed.

**Review by cston — APPROVED** [review]

**Review by cston — APPROVED** [review]

File: src/ExpressionEvaluator/CSharp/Test/ResultProvider/FullNameTests.cs [hunk 2]
```diff
@@ -1001,7 +1001,7 @@ public void MangledName_SimplifyBackingField_02()
 
             IDkmClrFullNameProvider2 fullNameProvider = new CSharpFormatter();
             var inspectionContext = CreateDkmInspectionContext();
-            Assert.Equal("<StringParameter>P", fullNameProvider.GetClrNameForField(inspectionContext, new DkmClrRuntimeInstance(assembly).Modules[0], fieldToken));
+            Assert.Equal("StringParameter", fullNameProvider.GetClrNameForField(inspectionContext, new DkmClrRuntimeInstance(assembly).Modules[0], fieldToken));
```
Comment (AlekseyTs) [inline src/ExpressionEvaluator/CSharp/Test/ResultProvider/FullNameTests.cs:1004]: >Assert.Equal("StringParameter", fullNameProvider.GetClrNameForField(inspectionContext, new DkmClrRuntimeInstance(assembly).Modules[0], fieldToken)); [](http://example.com/codeflow?start=12&length=148)

This behavior change feels unexpected. The scenario doesn't involve captured primary constructor parameters

**Pull request merged**